avivator/src/Avivator.js

```diff
@@ -180,7 +195,10 @@ export default function Avivator(props) {
     const mouseUp = event.type === 'mouseup';
     // Only update image on screen on a mouseup event for the same reason as above.
     if (mouseUp) {
-      const stats = await getChannelStats({ loader, loaderSelection });
+      const stats = await Promise.all(
+        loaderSelection.map(selection => getSingleSelectionStats({ loader, selection }))
+      );
+      console.log(stats)
       const domains = stats.map(stat => stat.domain);
       const sliders = stats.map(stat => stat.autoSliders);
```

Comment:
```suggestion
      const sliders = stats.map(stat => stat.slider);
```
This is why the z-stack was not working. `getSingleSelectionStats` returns `{ domain, slider }`